package.json

```diff
@@ -11,14 +11,16 @@
     "prebuild": "rimraf dist",
     "validate": "npm-run-all --parallel lint cover build test:configs --sequential check-coverage",
     "build": "babel -d dist src",
+    "dev": "npm-run-all --parallel watch:lint watch:test",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "report-coverage": "cat ./coverage/lcov.info | node_modules/.bin/codecov",
     "lint": "eslint .",
+    "watch:lint": "esw . -w",
     "cover": "nyc --reporter=lcov --reporter=text --reporter=html ava",
     "watch:cover": "nodemon --quiet --watch src --exec npm run cover -s",
-    "test": "ava",
-    "watch:test": "npm run test -- -w",
-    "test:configs": "ava tests/configs.js",
+    "test": "ava --verbose",
```

Comment:
I am bit puzzled,  are you preferring to pass the `--verbose` flag as part of the command instead of `ava config`?
>>I actually prefer not using verbose mode these days. If it's needed, you can use the flag